docs/tutorial/part-eight/index.md

```diff
@@ -1,94 +1,94 @@
+Wenn das Skript läuft, kannst du deine Seite unter `http://localhost:9000` betrachten.
 
-### Run a Lighthouse audit
+### Führe den Lighthouse Audit durch
 
-Now you're going to run your first Lighthouse test.
+Du führst jetzt deinen ersten Test mit Lighthouse durch.
 
-1.  If you haven't already done so, open the site in Chrome Incognito Mode so no extensions interfere with the test. Then, open up the Chrome DevTools.
+1.  Falls noch nicht geschehen, öffne deine Seite in einem Chrome Inkognito Fenster, sodass keine Chrome Extenstions den Test stören. Öffne danach deine Chrome DevTools.
 
-2.  Click on the "Audits" tab where you'll see a screen that looks like:
+2.  Klicke auf das "Audits" Tab, in dem du folgendes sehen solltest:
 
 ![Lighthouse audit start](./lighthouse-audit.png)
 
-3.  Click "Perform an audit..." (All available audit types should be selected by default). Then click "Run audit". (It'll then take a minute or so to run the audit). Once the audit is complete, you should see results that look like this:
+3.  Klicke auf "Perform an audit..." (Es sollten dabei standardmäßig alle Audit Typen ausgewählt sein). Klicke anschließend auf "Run audit". (Es dauert ca. eine Minute, bis der Audit fertig ist). Wenn der Audit fertig ist, solltest du deine Ergebisse sehen:
 
 ![Lighthouse audit results](./lighthouse-audit-results.png)
 
-As you can see, Gatsby's performance is excellent out of the box but you're missing some things for PWA, Accessibility, Best Practices, and SEO that will improve your scores (and in the process make your site much more friendly to visitors and search engines).
+Wie du sehen kannst, sind die Performance Ergebnisse von Gastby bereits "out of the box", d.h. ohne Optimierungen, ausgezeichnet. Dir fehlen allerdings einige Punkte in den Bereichen PWA, Accessibility (Barrierefreiheit), Best Practices und SEO um dein Gesamtergebnis noch zu steigern (und dabei deine Seite benutzer- und suchmaschinenfreundlicher zu machen).
 
-## Add a manifest file
+## Füge eine manifest Datei hinzu
 
-Looks like you have a pretty lackluster score in the "Progressive Web App" category. Let's address that.
+Es sieht so aus, als wären deine Ergebnisse im Bereich Progressive Web App (PWA) noch verbesserungswürdig. Lass uns das angehen.
 
-But first, what exactly _are_ PWAs?
+Aber zunächst: was _sind_ PWAs eigentlich _genau_?
 
-They are regular websites that take advantage of modern browser functionality to augment the web experience with app-like features and benefits. Check out [Google's overview](https://developers.google.com/web/progressive-web-apps/) of what defines a PWA experience.
+PWAs sind normale Webseiten, die sich moderne Browserfunktionen zu Nütze machen, um die Web Experience mit app-ähnlichen Features und Vorteilen zu ergänzen. Sieh dir dazu die [Übersicht auf Google](https://developers.google.com/web/progressive-web-apps/) an, um herauszufinden, was genau eine PWA ausmacht.
```

Comment:
```suggestion
PWAs sind normale Webseiten, die sich moderne Browserfunktionen zu Nütze machen, um die Web Experience mit App-ähnlichen Features zu ergänzen. Sieh dir dazu die [Übersicht auf Google](https://developers.google.com/web/progressive-web-apps/) an, um herauszufinden, was genau eine PWA ausmacht.
```

Is `zu Nütze` a proper translation? Or should it be `zu Nutze`?

Reply:
Good point. I couldn't really find `zu Nütze`, seems to be some kind of personal thing of mine 😅Let's go with `zu Nutze`.